files/internals/cports.common

```diff
@@ -118,20 +234,46 @@ PROTO="icmp"
 if [ "$ICMP_LIM" == "" ]; then
         ICMP_LIM=0
 fi
-if [ "$(echo $ICMP_LIM | tr '/' ' ' | awk '{print$1}')" -gt "0" ]; then
-        ICMP_EARGS="-m limit --limit $ICMP_LIM"
+if [ $EXECUTOR == "nft" ]; then
+   if [ "$(echo $ICMP_LIM | tr '/' ' ' | awk '{print$1}')" -gt "0" ]; then
+        ICMP_EARGS="limit rate $ICMP_LIM burst 5 packets"
```

Comment:
need to auto convert ICMP_LIM

Reply:
already https://github.com/jasonwee/advanced-policy-firewall/blob/39a094a17e20885d484e20fd35b86d8c4537feb3/files/internals/internals.conf#L93-L98